browser/src/canvas/sections/HTMLObjectSection.ts

```diff
@@ -33,7 +33,7 @@ class HTMLObjectSection extends CanvasSectionObject {
 		else this.sectionProperties.objectDiv.style.height = objectHeight + 'px';
 
 		if (extraClass)
-			this.sectionProperties.objectDiv.classList.add(extraClass);
+			this.sectionProperties.objectDiv.className += ' ' + extraClass;
```

Comment:
why? we will want multiple classes added or what is the reason here? I'm just curious

Reply:
Yes, this is for being able to provide multiple class names. The caller can send the extraClass like "class1 class2" etc.